source/_integrations/switchbee.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: SwitchBee
+description: Instructions for how to integrate SwitchBee accessories within Home Assistant.
+ha_category:
+  - Switch
+ha_release: 2022.10
+ha_iot_class: local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@jafar-atili'
+ha_domain: switchbee
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+[SwitchBee](https://www.switchbee.com), is an innovation company making smart homes more accessible and affordable to any household environment. 
+
+There is currently support for the following device types within Home Assistant:
+
+- Switch 
+- Timed Power Switch (Boiler)
+- Group Switch
+- Timed Switch
+
+Supported devices will be discovered after the `SwitchBee` integration is configured
+
+## Preliminary Steps
+
+You will need to provide the following parameters in order to set up the `SwitchBee` integration:
+
+- Central Unit IP
+- Username
+- Password
+- Initialize switches as light entities (False by default)
+
```

Comment:
These are asked by the config flow?

Reply:
Yes, its a Checkbox in the config flow.

I'd say 90% of the cases. their switches will be controlling actual lights.

For me personally it was a pain converting 76 switches to light in HA.

This is why I came up with this option.

Reply:
So, we don't need to describe config flow fields. The config flow should do that.

> I'd say 90% of the cases. their switches will be controlling actual lights.

That is native Home Assistant functionality, no need for an integration to implement its own logic. It needs to be removed from the documentation and the core integration before the next release.